File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    p_count=0
    y_count=0
    for i in s:
        print(i)
        if i=='p' or i=='P':
            p_count+=1
        elif i=='y' or i=='Y':
            y_count+=1
        

    return True if p_count==y_count else False

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int p=0,y=0;
        
        for(int i=0;i<s.length();i++){
            char ch=s.charAt(i);
            if(ch=='p' || ch=='P') p++;
            if(ch=='y' || ch=='Y') y++;
        }
        
        if(p==y) return true;
        else return false;
    }
}

/*
대문자로 통일하기 -> 굳이?
int p
int y

각 카운트 계산하기 -> 합계 비교
*/